buildbot/Makefile

```diff
@@ -43,7 +43,7 @@ test: setup test_normal test_security
 test_normal:
 	@echo "Testing Riak Python Client (without security)"
 	@$(RIAK_ADMIN) security disable
-	@RIAK_TEST_PROTOCOL='pbc' RIAK_TEST_PB_PORT=8087 RUN_POOL=1 RUN_BTYPES=1 RUN_CLIENT=1 RUN_MAPREDUCE=1 RUN_KV=1 RUN_RESOLVE=1 RUN_YZ=1 RUN_DATATYPES=1 RUN_INDEXES=1 ./tox_runner.sh ..
```

Comment:
No, I added `RUN_POOL` a commit or so ago just for kicks. No need to run them every time. And, they don't have anything to do with the multi-pools, anyway, they are tests of a the generic `Pool` class that the TCP and HTTP connection pools use.